go.mod

```diff
@@ -1,16 +1,14 @@
 module github.com/operator-framework/operator-lifecycle-manager
 
-go 1.23.0
-
-toolchain go1.23.4
+go 1.23.6
```

Comment:
@tmshort @grokspawn ^ that can work for us.
I changed it because it was set to 1.23.3 above, and I'm not sure if we have this one.
So, since 1.23.6, we know that we are safe. I moved to this one.